### PR TITLE
added changes for cancel query not being able to cancel, added back ticks

### PR DESCRIPTION
**Pull request by sumukhswamy**
### Description
cancel async query for running query
added backtiskc for sample and other queries

 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by RyanL1997 — COMMENTED** [review]
Hi @sumukhswamy , thanks for putting this together! The change itself LGTM. Could you also fix the Lint? I know it is kinda irrelevant from this change but since the lint rules are targeting on the change files so.

File: public/components/Main/main.tsx [hunk 1]
```diff
@@ -259,7 +259,7 @@ export class Main extends React.Component<MainProps, MainState> {
       isResultFullScreen: false,
       selectedDatasource: [{ label: 'OpenSearch' }],
       asyncLoading: false,
-      asyncLoadingStatus: 'SUCCESS',
+      asyncLoadingStatus: 'success',
```
Comment (mengweieric) [inline public/components/Main/main.tsx:262]: Maybe use enum?

File: public/components/Main/main.tsx [hunk 2]
```diff
@@ -533,8 +533,8 @@ export class Main extends React.Component<MainProps, MainState> {
       const result: ResponseDetail<string> = this.processQueryResponse(
         response as IHttpResponse<ResponseData>
       );
-      const status = result.data.status;
-      if (_.isEqual(status, 'SUCCESS')) {
+      const status = result.data.status.toLowerCase();
```
Comment (mengweieric) [inline public/components/Main/main.tsx:536]: any specific reason for converting to lower case? Can't we compare directly?

File: public/components/SQLPage/table_view.tsx [hunk 3]
```diff
@@ -139,7 +139,7 @@ export const TableView = ({ http, selectedItems, updateSQLQueries, refreshTree }
       setTableNames([]);
       const query = {
         lang: 'sql',
-        query: `SHOW SCHEMAS IN ${selectedItems[0].label}`,
+        query: `SHOW SCHEMAS IN \`${selectedItems[0]['label']}\``,
```
Comment (mengweieric) [inline public/components/SQLPage/table_view.tsx:142]: do we need to add guards for selectedItems?

File: public/components/Main/main.tsx (hunk 2 above)
Comment (sumukhswamy) [reply public/components/Main/main.tsx:536]: wanted to keep it uniform if there are other changes, initially the statuses from the back end were in caps

File: public/components/SQLPage/table_view.tsx (hunk 3 above)
Comment (sumukhswamy) [reply public/components/SQLPage/table_view.tsx:142]: yes they are its being fetched from the main page

**Review by paulstn — APPROVED** [review]
lgtm

File: public/components/Main/main.tsx (hunk 1 above)
Comment (paulstn) [reply public/components/Main/main.tsx:262]: +1, would you be able to use AsyncQueryLoadingStatus for this?

**Review by Swiddis — APPROVED** [review]

**Pull request merged**